### PR TITLE
GeoServer 2.19.0 release page and announcement 

### PR DESCRIPTION
**Pull request by jodygarnett**
As this is the first release of 2.19.x there are some additional steps:

* [x] release announcement
* [x] 2.19.0 release page
* [x] update stable, maintenance, dev pages
* [x] fix download mark 2.19 as stable, 2.18 as maintenance,  move 2.17 as archived
* [x] address some broken 2.17 download links mentioned by @jnh5y

### CODE REVIEW

**Comment by jodygarnett** [comment]
@aaime thanks for your help last week, I started the website update and blog post to hopefully give you some of your weekend :)

File: _posts/2021-03-22-geoserver-2-19-0-released.md [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+
+GeoServer 2.19.0 includes the latest JTS Topology Suite 1.18.1 release.
+
+Thanks to Martin Davis (Crunch Data) and James Hughes (CCRi) for making [JTS 1.18.1](https://github.com/locationtech/jts/releases/tag/jts-1.18.1) available  during our release window.
+
+## Codebase updates and Quality Assurance
+
+GeoServer continues to be build with the latest open source technologies:
+
+* GeoTools 26.0
```
Comment (mprins) [inline _posts/2021-03-22-geoserver-2-19-0-released.md:208]:
```suggestion
* GeoTools 25.0
```

File: _posts/2021-03-22-geoserver-2-19-0-released.md [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+* ``DownloadMapProcess``, allows to download a large map matching what is visible on a client (which may be using tiles and display on a multi-screen), eventually dynamically fetching layers from remote WMS servers as well. It's also possible to decorate the final map using the standard [decoration layouts](https://docs.geoserver.org/latest/en/user/services/wms/decoration.html#wms-decorations).
+* ``DownloadAnimationProcess``, allows to build a MP4 movie given a set of layers and times.
+
+GeoNode uses the module to allow download of datasets, eventually clipped and filtered to the current view. The asynchronous download allows
+to download large datasets, and retrieve them later, once ready.
+
+![GeoNode download](/img/posts/2.19-RC/geonode_download_1.png)<br/>
+*Initiating a download in GeoNode*
+
+![GeoNode download](/img/posts/2.19-RC//geonode_download_2.png)<br/>
```
Comment (bradh) [inline _posts/2021-03-22-geoserver-2-19-0-released.md:97]: Maybe don't want the `//` in the URL.

File: _posts/2021-03-22-geoserver-2-19-0-released.md [hunk 3]
```diff
@@ -0,0 +1,259 @@
...
+``
+
+For more information, refer to the [module documentation](https://docs.geoserver.org/stable/en/user/extensions/params-extractor/index.html).
+
+Thanks to Nuno Oliveira (GeoSolutions) for developing this module.
+
+
+## GeoWebCache-S3 extension
+
+The GeoWebCache S3 blobstore allows to store GeoWebCache tiles in a S3 bucket. It has been also tested with a few other S3 compatible blob
```
Comment (bradh) [inline _posts/2021-03-22-geoserver-2-19-0-released.md:171]: "allows to store" -> "allows storing"

File: _posts/2021-03-22-geoserver-2-19-0-released.md [hunk 4]
```diff
@@ -0,0 +1,259 @@
...
+The Script community module has been retired.
+
+The module provided scripting abilities for GeoServer, allowing to add WPS processes and small REST services in scripting languages, 
+and storing them in the data directory. 
+
+Unfortunately the module fell un-maintained and would no longer build nor work.
+
+## JTS 1.18.1
+
+GeoServer 2.19.0 includes the latest JTS Topology Suite 1.18.1 release, the headline feature is an optional "Overlay Next Generation" implementation that should provide a performance improvement for things tile generation, vector tiles, and get map requests.
```
Comment (bradh) [inline _posts/2021-03-22-geoserver-2-19-0-released.md:200]: Maybe replace "things" with "operations such as"

File: _posts/2021-03-22-geoserver-2-19-0-released.md [hunk 5]
```diff
@@ -0,0 +1,259 @@
...
+
+Unfortunately the module fell un-maintained and would no longer build nor work.
+
+## JTS 1.18.1
+
+GeoServer 2.19.0 includes the latest JTS Topology Suite 1.18.1 release, the headline feature is an optional "Overlay Next Generation" implementation that should provide a performance improvement for things tile generation, vector tiles, and get map requests. 
+
+To try it out use the system property ``-Djts.overlay=ng`` - the effect should be small as we already have several optimizations in place before trying this now faster JTS Overlay.
+
+Thanks to Martin Davis (Crunch Data) and James Hughes (CCRi) for making [JTS 1.18.1](https://github.com/locationtech/jts/releases/tag/jts-1.18.1) available  during our release window.
```
Comment (bradh) [inline _posts/2021-03-22-geoserver-2-19-0-released.md:204]: Crunchy Data?

**Review by bradh — APPROVED** [review]

**Pull request merged**